pandas/_libs/lib.pyx

```diff
@@ -1079,6 +1080,7 @@ _TYPE_MAP = {
     "timedelta64[ns]": "timedelta64",
     "m": "timedelta64",
     "interval": "interval",
+    Period: "period",
```

Comment:
prob need Interval here?

Reply:
Interval is already handled by the "interval" string in the line above, see the non-inline comment with the link to PR were this was fixed before